Thirdlesson/GeometricTransformations/GeometricTransformations.cpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by s1000633 on 5/18/21.
+//
+#include "TransformImage.h"
+using namespace cv;
+
+int main() {
+    //Load image
+    Mat image = cv::imread("photo.jpg");
+    TransformImage transformImage(image);
+    int cols = image.cols * 2;
+    int rows = image.rows * 2;
+
+    transformImage.resize_image(cols, rows);
+
+    int angle = 90;
```

Comment:
can be constexpr

Reply:
angle?

Reply:
yes, any comment I left related to code line above it